react-ui/src/components/LogInForm.test.js

```diff
@@ -0,0 +1,230 @@
+    const password = getByPlaceholderText(dom.container, 'password');
+    const submit = getByDisplayValue(dom.container, "Log In!");
+    const expectedArg = "Invalid Email or Password";
+    const resp = { email: 'joshuabrooks@gmail.com', password: 'post' }
+    await axios.get.mockImplementation(() => Promise.resolve(resp));
+
+    // Act
+    window.alert = jest.fn();
+    fireEvent.change(email, { target: { value: 'joshuabrooks@gmail.com' } });
+    fireEvent.change(password, { target: { value: 'post' } });
+    fireEvent.click(submit);
+
+    //Assert
+    await axios.get
+    expect(window.alert).toHaveBeenCalledWith(expectedArg);
+
+  });
+
+  test('testing valid input for API call', async () => { // cover lines
+    //Arrange
+    const dom = render(
+      <BrowserRouter>
+        <LogInForm />
+      </BrowserRouter>);
+    const email = getByPlaceholderText(dom.container, 'email');
+    const password = getByPlaceholderText(dom.container, 'password');
+    const submit = getByDisplayValue(dom.container, "Log In!");
+    const resp = { data: true }
+    await axios.get.mockImplementation(() => Promise.resolve(resp));
+
+    // Act
+    window.alert = jest.fn();
+    fireEvent.change(email, { target: { value: 'joshuabrooks@gmail.com' } });
+    fireEvent.change(password, { target: { value: 'j1234' } });
+    fireEvent.click(submit);
+
+    //Assert
+    expect(axios.get).toHaveBeenCalled();
+  });
+
+
+  test('loading of dashboard page from log in', async () => { // doesn't cover statement
+    //Arrange
+    const { rerender, getByPlaceholderText, getByDisplayValue, queryByPlaceholderText } = render(
+      <BrowserRouter>
+        <LogInForm />
+      </BrowserRouter>);
+
+    const testWorker = [{
+      accessToken: "eeJAQr3wEC6CJZROFJTY",
+      email: "joshuabrooks@gmail.com",
+      firstName: "Joshua",
+      islogged: true,
+      password: 'j1234',
+      userType: "worker",
+      workerId: 8000000
+    }]
+
+    const email = getByPlaceholderText('email');
+    const password = getByPlaceholderText('password');
+    const submit = getByDisplayValue("Log In!");
+
+
+    // Act
+    fireEvent.change(email, { target: { value: 'joshuabrooks@gmail.com' } });
+    fireEvent.change(password, { target: { value: 'j1234' } });
+
+    const resp = { data: true };
+    await axios.get.mockImplementation(() => Promise.resolve(resp));
+    rerender(<BrowserRouter>
+      <LogInForm loginParams={testWorker} />
+    </BrowserRouter>);
+
+    fireEvent.click(submit);
+      
+    //Assert
+    rerender(<BrowserRouter>
+      <Route exact path="/login" render={props => (<Redirect to={`/dashboard/${'joshuabrooks@gmail.com'}/${'worker'}/${'Joshua'}/${8000000}/${'eeJAQr3wEC6CJZROFJTY'}`} Component={Dashboard} />)}></Route>
+    </BrowserRouter>)
+    expect(queryByPlaceholderText('email')).toBeNull();
+  });
+
+  /*  test('testing valid email and password with set changes', async () => {
```

Comment:
Hi Kyle,
I would suggest adding a comment as to why this test was removed or just to remove this test altogether to avoid future confusion.

Reply:
removed unneeded test